lib/file_store.ex

```diff
@@ -35,14 +39,18 @@ defprotocol FileStore do
   Write a file to the store. If a file with the given `key`
   already exists, it will be overwritten.
 
+  ## Options
+
+      * `:content_type` - Sets the content type hint for the adapter.
```

Comment:
I think the doc for `:disposition` is missing here.

Reply:
Done